### PR TITLE
Layout styling fixes

### PR DESCRIPTION
**Pull request by Vadko**
## Description

* Added ListComponentStyleProp to provide header styles
* Moved style prop from scrollview to view that wraps items in list
* Added contentContainerStyle prop
* Renamed prop for ScrollView from style to containerStyle

## Related Issues

I included props to style the component header and renamed style prop on the scrollView, due to the problems it caused when using styles for whole component for header component to containerStyle. Also, I've added contentContainerStyle and wrapper header with View to provide styles for it.

## Tests

I added the following tests:

Test Suites: 1 passed, 1 total
Tests:       10 passed, 10 total
Snapshots:   1 passed, 1 total
Time:        2.596 s

## Checklist

- [x] I read the [Contributor Guide](https://github.com/hyochan/react-native-masonry-list/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn lint && yarn tsc`
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: index.tsx [hunk 1]
```diff
@@ -84,15 +92,22 @@ function MasonryList<T>(props: Props<T>): ReactElement {
         if (isCloseToBottom(nativeEvent, onEndReachedThreshold || 0.1))
           onEndReached?.();
       }}>
-      {ListHeaderComponent}
+      <View style={ListHeaderComponentStyle}>{ListHeaderComponent}</View>
```
Comment (hyochan) [inline index.tsx:95]: I think this is `redundant`. Users can have more freedom to create components on the other side.

**Review by hyochan — CHANGES_REQUESTED** [review]

File: index.tsx (hunk 1 above)
Comment (Vadko) [reply index.tsx:95]: The problem is when we want to apply style only for list, not for the header - we can't do that, thats why I think this props is should be, actually, for the same reason original FlatList in RN have this prop https://reactnative.dev/docs/flatlist#listheadercomponentstyle

Comment (hyochan) [reply index.tsx:95]: Thanks for the pointer! Totally understandable!

**Review by hyochan — APPROVED** [review]
I've changed snapshot tests for you and will merge this!

**Pull request merged**